Code/ForceField/MMFF/testMMFFForceField.cpp

```diff
@@ -1216,7 +1224,10 @@ int mmffValidationSuite(int argc, char *argv[]) {
                     ForceFields::MMFF::isDoubleZero(refTorsionEnergy))) {
                 fixTorsionInstance(torsionInstance);
                 refTorsionInstanceVec.push_back(torsionInstance);
-              }
+              } else {
+                delete torsionInstance;
+              torsionInstance;
```

Comment:
Why is this here?

Reply:
Oops... that "torsionInstance;" slipped in, somehow. I'll remove it.